### ISSUE TITLE
Connection refused : http://exac.hms.harvard.edu/

### ISSUE THREAD

**Issue by gpcr** [hms-dbmi/exac.hms.harvard.edu#3]
exac Api access database down?

**Comment by IbrahimTanyalcin** [comment]
Yes I think so, it was down yesterday (2018-06-08), the website went back up but the requests still return 404.

**Comment by gpcr** [comment]
rest API still down:  "There was an error in communicating with the server. Please try again later."
